components/provisioning/warewulf-provision/SPECS/warewulf-provision.spec

```diff
@@ -203,7 +209,7 @@ image and to provide boot capability for %{_arch} architecture.
 Summary: Warewulf - System provisioning server
 Requires: %{name} = %{version}-%{release}
 Requires: %{name}-server-ipxe-%{_arch} = %{version}-%{release}
-Requires: %{httpsvc}, perl(Apache), %{tftpsvc}, %{dhcpsrv}
+Requires: %{httpsvc}, %{perl_apache}, %{tftpsvc}, %{dhcpsrv}, tcpdump
```

Comment:
seems noneed.

1. Add OpenHPC on openEuler extra yum source:
```
# cat /etc/yum.repos.d/ohpc-extra.repo
[OpenHPC_oepkg]
name=OpenHPC OEPKG
type=rpm-md
baseurl=https://repo.oepkgs.net/openEuler/rpm/openEuler-22.03-LTS/contrib/OpenHPC/aarch64
gpgcheck=0
enabled=1
```

2. Build and install ohpc-warewulf
```
tests/ci/run_build.py ohpc components/admin/meta-packages/SPECS/meta-packages.spec
cd /home/ohpc/rpmbuild/RPMS/aarch64/
yum install ./ohpc-warewulf-2.6.1-19999.ci.ohpc.aarch64.rpm
```

It seems `perl(Apache)` is equal to `mod_perl`.

Reply:
OK! Let's add this extra repo to the OBS configuration! @adrianreber Could you please help us with this ?

Reply:
> It seems perl(Apache) is equal to mod_perl.

I can understand they are the same thing but I cannot understand how RPM could do the mapping.
The new repo contains https://repo.oepkgs.net/openEuler/rpm/openEuler-22.03-LTS/contrib/OpenHPC/aarch64/Packages/mod_perl-2.0.12-1.aarch64.rpm and its .spec does not mention `perl-Apache`

Reply:
try: `rpm -qp mod_perl-2.0.12-1.aarch64.rpm --provides | grep Apache`

Reply:
Magic!
```
rpm -qp mod_perl-2.0.12-1.aarch64.rpm --provides | grep Apache
perl(Apache)
perl(Apache2::Access) = 2.000012
perl(Apache2::CmdParms) = 2.000012
perl(Apache2::Command) = 2.000012
...
```
I will remove the extra logic in this PR!

Reply:
Thank you, @adrianreber !